_data/speakers.yml

```diff
@@ -15,14 +17,137 @@
     - icon: "slides"
       title: "hannelita"
       link: "http://slides.com/hannelitavante-hannelita"
-- name: "Felix Klock"
+felix:
+  name: "Felix Klock"
+  featured: true
   nick: "pnkfelix"
   desc: "Felix Klock is a research engineer at Mozilla, where he works on the Rust compiler, runtime libraries, and language design. He previously worked on the ActionScript Virtual Machine for the Adobe Flash runtime. Felix obtained his PhD in Computer Science from Northeastern University, and is also one of the developers of the Larceny Scheme language runtime."
   thumbnailUrl: felix.jpg
   links:
     - icon: "twitter"
       title: "pnkfelix"
-      link: "https://github.com/pnkfelix"
+      link: "https://twitter.com/pnkfelix"
     - icon: "github"
-      title: "hannelita"
+      title: "pnkfelix"
       link: "https://github.com/pnkfelix"
+sonja:
+  name: "Sonja Heinen"
+  nick: sonjaheinen
+  featured: true
+  desc: "Sonja is a semi-nomadic visual designer interested in the space where technology, art and society collide. In October 2014 she earned a Master‘s in Sustainable Design from Kingston University in London, then relocated to Berlin. She currently works as a Frontend developer and roams through open source communities."
```

Comment:
Master's
Master's Degree
Masters Degree

I'm never sure what is the correct thing to use